### ISSUE TITLE
Ledger receive address is null

### ISSUE THREAD

**Issue by Thoralf-M** [iotaledger/firefly#1266]
## Issue description

Ledger receive address is null and there is no way to close the popup. Ledger never showed the prompt

## Wallet version

- wallet version (version number, commit, or branch): (Ledger Migration test build) https://github.com/iotaledger/firefly/actions/runs/1040500341

## System specification

Win 10

## Is it a repeatable issue?

N

## Steps To reproduce the issue

1. I generated a new account (that already had history)
2. As soon as receive address button was available I clicked on it

## Expected behaviour

It should show the address and prompt on the ledger

## Actual behaviour

Displayed null and showed an error very short, but not sure which one it was

## Errors

```


Mon, 19 Jul 2021 08:56:20 GMT
ClientError: `Response error with status code 404: {"error":{"code":"404","message":"Not Found, error: output not found: dc286e83c1d38b1469cbc302f0077617916bb4ef71b4b60ae20be42b844da6c50000: code=404, message=Not Found"}}
`

Mon, 19 Jul 2021 08:50:32 GMT
ClientError: {}

```

![image](https://user-images.githubusercontent.com/46689931/126133477-ef2ce6e7-df4e-481c-9d1a-d613100a07e8.png)

**Comment by laumair** [comment]
Probably only specific to accounts with history 

/cc @maxwellmattryan (since you decided to pick this up)

**Comment by cvarley100** [comment]
Fixed in #1271
